.github/workflows/test-job.yaml

```diff
@@ -397,33 +397,13 @@ jobs:
           docker.io/${{env.REPO_ORG}}/${{env.REPO_NAME}}:nightly
       if: contains(fromJson('["refs/heads/master"]'), github.ref)
 
-  test-compatibility-2_5_0:
-    needs:
-      - build_operator
-      - build_apiserver
-      - lint
-    runs-on: ubuntu-latest
-    name: Compatibility Test - 2.5.0
-    steps:
-      - name: Check out code into the Go module directory
-        uses: actions/checkout@v2
-        with:
-          # When checking out the repository that
-          # triggered a workflow, this defaults to the reference or SHA for that event.
-          # Default value should work for both pull_request and merge(push) event.
-          ref: ${{github.event.pull_request.head.sha}}
-
-      - uses: ./.github/workflows/actions/compatibility
-        with:
-          ray_version: 2.5.0
-
-  test-compatibility-2_6_3:
```

Comment:
Previously we were testing 2.5.0, 2.6.3, 2.7.0, and 2.8.0.  

In this PR, we only test 2.7.0, 2.8.0, and 2.9.0.